samcli/commands/local/lib/route_collector.py

```diff
@@ -0,0 +1,59 @@
+"""
+Class to store the Route list in the SAM Template. This class helps store both implicit and explicit
+routes in a standardized format
+"""
+
+import logging
+from collections import defaultdict
+
+LOG = logging.getLogger(__name__)
+
+
+class RouteCollector(object):
```

Comment:
Was this a merge/rebase mistake? It doesn't appear that RouteCollector is being used.